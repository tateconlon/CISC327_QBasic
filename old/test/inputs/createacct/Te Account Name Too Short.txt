login
agent
createacct
2234567
Te
logout