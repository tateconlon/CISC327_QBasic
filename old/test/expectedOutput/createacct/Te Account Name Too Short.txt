login as agent successful
Te is not a valid account name
Transaction summary written and system logged out successfully